src/main/resources/static/data-feedback.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Created by bo on 5/7/16.
+ */
+
+d3.json("data/test-feedback-data.json", function (data) {
+    console.log(data);
+    for(var i=0; i<data.length; i++) {
+        var fb = data[i];
+        var msg = fb['message'];
+        var vals = fb['offendingValues'];
+        var msgRow =
+            '<tr>' +
+            '<th scope="row">Message: </th>' +
+            '<td>  </td>' +
+            '</tr>' +
+            '<tr>' +
+            '<th scope="row">  </th>' +
+            '<td>'+msg+'</td>' +
+            '</tr>';
+        var valRows = '<tr><th scope="row">offendingValues: </th><td></td></tr>';
+
+        for(var j=0; j<vals.length; j++) {
```

Comment:
If you press ALT+CMD + L IntelliJ will format nicely your code. 
Right now there are a lot of little inconsistencies and styling mistakes, most of them will be handled just by this one key-combination.